.github/BOTMETA.yml

```diff
@@ -1423,6 +1423,9 @@ files:
     ignore: matze
     labels: zypper
     maintainers: $team_suse
+  $modules/zendesk_ticket.py:
```

Comment:
We try and keep this in alphabetical order. Could you please move it to before the zfs modules? Thx